CHANGELOG.md

```diff
@@ -20,58 +20,47 @@ following sections:
 
 The versions follow [semantic versioning](https://semver.org).
 
-## Unreleased - YYYY-MM-DD
-
-### Changed
-### Deprecated
-### Removed
-### Fixed
-### Security
-### Added
-
-## 3.1 - 2023-06-21
+## 3.2 - YYYY-MM-DD
```

Comment:
I assume the template needs to be kept as convenience for the future:

```suggestion
## Unreleased - YYYY-MM-DD

### Changed
### Deprecated
### Removed
### Fixed
### Security
### Added

## 3.2 - YYYY-MM-DD
```

Reply:
Uuuuugh.

Let's switch to [Protokolo](https://codeberg.org/carmenbianca/protokolo) after this release. I am so tired of fighting with change logs.

Fixing this.